test/suite/test_compat01.py

```diff
@@ -45,16 +45,16 @@ class test_compat01(wttest.WiredTigerTestCase, suite_subprocess):
     # Declare the log versions that do and do not have prevlsn.
     # Log version 1 does not have the prevlsn record.
     # Log version 2 introduced that record.
-    # Log version 3 continues to have that record.
-    # Log version 4 continues to have that record.
+    # Log versions 3-5 continue to have that record.
```

Comment:
Ah, this is pretty much what I said in the other review. You could generalize it by just saying "Log versions 3 and later..." so that we don't need to keep updating it.

Reply:
Done.